src/pages/patientView/mutationalSignatures/MutationalSignaturesContainer.tsx

```diff
@@ -34,10 +34,11 @@ export default class MutationalSignaturesContainer extends React.Component<
         // split the id by "_", the last part is the version info
         // we know split will always have results
         // use uniq function to get all unique versions
-        return _.chain(this.props.profiles)
+        // not all patients have the newest mutational signatures --> check if data is present before giving the options
+        const possibleOptions = this.props.profiles
```

Comment:
the original code (using lodash chaining is easier to read).  you should try to retain that style.

Reply:
Retained the style and rewrote the lines of code to:
`_.chain(this.props.profiles)
            .map(profile => _.last(profile.molecularProfileId.split('_'))!).filter(item => item in this.props.data).uniq().value();`